layers/+tools/eaf/packages.el

```diff
@@ -131,6 +131,7 @@
               ("-" . "insert_or_zoom_out")
               ("=" . "insert_or_zoom_in")
               ("0" . "insert_or_zoom_reset")
+              ;; ("d" . "insert_or_dark_mode")
```

Comment:
I thought I do keep it here. It can be made working again with some modificatios to the eaf package (the same modifications I made there before)